.env.example

```
@@ -0,0 +1,15 @@
+# Neon Database
+DATABASE_URL=postgresql://user:pass@host/db
+
+# Cloudflare R2
+R2_ACCESS_KEY_ID=
+R2_SECRET_ACCESS_KEY=
+R2_BUCKET_NAME=
+R2_ENDPOINT=
+
+# Modal API
+MODAL_TOKEN_ID=
+MODABL_TOKEN_SECRET=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MODABL_TOKEN_SECRET key should go before the MODAL_TOKEN_ID key

(UnorderedKey)